django_freeradius/base/models.py

```diff
@@ -398,12 +398,30 @@ class AbstractRadiusAccounting(models.Model):
                                        blank=True)
     framed_ip_address = models.GenericIPAddressField(verbose_name=_('framed IP address'),
                                                      db_column='framedipaddress',
-                                                     db_index=True,
                                                      # the default MySQL freeradius schema defines
                                                      # this as NOT NULL but defaulting to empty string
                                                      # but that wouldn't work on PostgreSQL
                                                      null=True,
                                                      blank=True)
+    framed_ipv6_address = models.GenericIPAddressField(verbose_name=_('framed IPv6 address'),
+                                                       db_column='framedipv6address',
+                                                       null=True,
+                                                       blank=True)
```

Comment:
the ipv6 fields should have GenericIPAddressField.protocol set to ipv6:
https://docs.djangoproject.com/en/3.0/ref/models/fields/#genericipaddressfield